### DISCUSSION TITLE
Why showing the preheating and the post-shot data in the extraction plot?

### DISCUSSION THREAD

**Discussion by NilsBruch** [obiwan007/despresso#227] [Ideas]
Hello everyone, 

I would like to know why the 'preheat' and 'post-shot' phases are shown in the extraction plot? Also, I would like to suggest that the "preheat" step should not be editable for each profile, but should be a setting in the app. This would simplify the profile structure significantly. What do you think?

Best
Nils

**Comment by tadelv** [comment]
<img width="1210" alt="Screenshot 2024-11-03 at 19 15 08" src="https://github.com/user-attachments/assets/5a6ab804-004d-46c1-918f-2e730564b7c3">
@NilsBruch is this the setting you are looking for? This will disable showing the pre-heat graph, you can find it in settings -> coffee pouring.
As for the post-shot phase, I don't believe there is a setting for this and it might get included, depending on if the machine decides to start a stabilization phase after the shot pour ends.

**Reply by NilsBruch** [reply]
I tried this setting. It switches off the showing of both the heating and the post-shot phase. This is in principle good. However, as I see it, with this setting turned on off, the weight of the coffee that drips into the cup after extraction is not counted. Therefore, my recipe diary shows that the extractions stopped too early, while in reality the SAW setting is correct. I think it would be better if this setting would be only "visual". What do you think?

However, now that I am using the app more often, I have to say that I am getting used to seeing pre-heating and post-shot data, so this is not really a problem for me personally.

**Reply by tadelv** [reply]
Yes, same here. I prefer seeing the pre-heat graph as it helps me understand what the machine is doing and why no coffee is coming out at that time :) I guess it would be nice to see that data, but not include it in the graphs when the shot is complete.
